ironfish-cli/src/commands/service/bridge/relay.ts

```diff
@@ -213,6 +221,10 @@ export default class BridgeRelay extends IronfishCommand {
       await api.sendBridgeDeposits(sends)
     }
 
```

Comment:
We are not account for sends with multiple asset types, perhaps we just say this is currently not supported. It would be possible given the standard transaction construction, but would result in a weird state.

Reply:
I think not supported for now -- we'd need a new wrapped asset on the Ethereum side for each custom asset that we want to support bridging.